spec/models/guide_spec.rb

```diff
@@ -288,3 +288,90 @@
     expect(guide.live_edition).to eq(nil)
   end
 end
+
+RSpec.describe Guide, "#in_state" do
```

Comment:
Scopes are (eventually) class methods so should read `".in_state"`. Same for the describe blocks below.